starboard/build/config/BUILDCONFIG.gn

```diff
@@ -313,6 +313,90 @@ template("copy") {
   }
 }
 
+template("evergreen_loader") {
+  # Add additional evergreen_loader target for every native target.
+  original_target_name = invoker.original_target_name
+  data_deps = invoker.data_deps
+  data_deps += [ "//third_party/icu:icudata" ]
```

Comment:
This is not within a target, it's in a template. It should be renamed or used only locally in targets in which it is needed